src/ed_gateway/application/contracts/infrastructure/api/abc_api.py

```
@@ -12,3 +14,6 @@ def core_api(self) -> ABCCoreApiClient: ...
     @property
     @abstractmethod
     def auth_api(self) -> ABCAuthApiClient: ...
+
+    @property
+    def notification_api(self) -> ABCNotificationApiClient: ...
```

Ruff (0.11.9)

`ABCApi.notification_api` is an empty method in an abstract base class, but has no abstract decorator

(B027)